### PR TITLE
SRE-2998 - Adding common label for grabbing grafana dashboards

### PR DESCRIPTION
**Pull request by sivakathali**
**Relevant Infra PR:**
https://github.com/Bluescape/infrastructure/pull/7216

### CODE REVIEW

File: charts/bluescape-banzai-dashboards/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.2
+version: 0.1.3-alpha
```
Comment (ashwinrajadurai3) [inline charts/bluescape-banzai-dashboards/Chart.yaml:18]: Not here

File: charts/bluescape-monitoring-dashboards/Chart.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.26
+version: 0.3.27-alpha
```
Comment (ashwinrajadurai3) [inline charts/bluescape-monitoring-dashboards/Chart.yaml:18]: This alone works

**Review by ashwinrajadurai3 — CHANGES_REQUESTED** [review]

File: charts/bluescape-banzai-dashboards/Chart.yaml (hunk 1 above)
Comment (sivakathali) [reply charts/bluescape-banzai-dashboards/Chart.yaml:18]: But in this chart also we made changes

Comment (ashwinrajadurai3) [reply charts/bluescape-banzai-dashboards/Chart.yaml:18]: That's fine, this isn't referenced in infra repo

**Review by ashwinrajadurai3 — CHANGES_REQUESTED** [review]

**Review by notjames — DISMISSED** [review]
please see comments in https://github.com/Bluescape/infrastructure/pull/7216

**Review by alexeyatbluescape — APPROVED** [review]
LGTM!

**Review by ashwinrajadurai3 — APPROVED** [review]

**Review by ashwinrajadurai3 — APPROVED** [review]
LGTM

**Pull request merged**